src/wavesurfer.js

```diff
@@ -573,12 +573,12 @@ const WaveSurfer = util.extend({}, util.observer, { util }, {
         );
     },
 
-    getArrayBuffer: function (url, callback) {
+    getArrayBuffer(url, callback) {
         var my = this;
 
         var ajax = WaveSurfer.util.ajax({
-            url: url,
-            responseType: 'arraybuffer'
```

Comment:
I changed the eslint rule to apply to methods, property shorthands are not enforced and I reverted this code to be like it was.